packages/common/src/ui/layout/tables/context/TableContext.tsx

```diff
@@ -42,7 +42,7 @@ const TableProvider = <T extends RecordWithId>({
 }: PropsWithChildren<{
   initialStore?: UseBoundStore<TableStore>;
   createStore: () => UseBoundStore<TableStore>;
-  queryParamsStore?: UseBoundStore<QueryParamsState<T>>;
+  queryParamsStore: UseBoundStore<QueryParamsState<T>>;
```

Comment:
thought that I'd make this required to prevent potential errors when calling the hook